gmail-to-salesforce-lead/README.md

```diff
@@ -0,0 +1,80 @@
+# Example to create leads obtained via email on Salesforce using the OpenAI API.
+
+## Use case
+The following sample demonstrates a scenario in which customer leads obtained through email are automatically pushed to Salesforce. The required details for the lead (name, company, designation etc.) are inferred using the content of the email and the OpenAI chat API.
+
+When the user receives an email pertaining to a lead, they will mark that thread with a specific label (e.g. `"Lead"`). This Ballerina program will run continuously in the background, polling the email server every 10 minutes for threads marked with this label. If an email is found, its content will be read and used to infer the following details
+* First name
+* Last name
+* Phone number
+* Email address
+* Company name
+* Designation
+
+Once these details have been inferred, a new lead will be generated on Salesforce.
+
+![Flow diagram](/docs/flow-diagram.png)
+
+## Prerequisites
+* An email account configured to use Gmail
+* An account on the Google Cloud Platform
+* An OpenAI account with API usage enabled
+* A Salesforce account
+
+>Note: The following steps will require you to generate keys for the Gmail, OpenAI and Salesforce APIs. These keys will have to be securely stored in the `Config.toml` file in the project directory under the relevant fields. To see a template of this file, please see the [`Config.toml.example`] file (./Config.toml.example).
+
+### Configuring your email account to use Gmail
+> Note: If you already have a Gmail account (ending with `@gmail.com`) or your acccount is on the Google workspace, you do not need to follow the steps below. In essence, if you can access your email via `www.gmail.com`, the following is not necessary.
```

Comment:
```suggestion
> Note: If you already have a Gmail account (ending with `@gmail.com`) or your account is on the Google workspace, you do not need to follow the steps below. In essence, if you can access your email via `www.gmail.com`, the following is not necessary.
```

Reply:
Fixed in [a718fba](https://github.com/ballerina-guides/integration-samples/pull/32/commits/a718fba876d7ed04e212bd0cd384c113178380cb)